tests/unit/materializers/test_pillow_image_materializer.py

```diff
@@ -0,0 +1,57 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import os
+from contextlib import ExitStack as does_not_raise
+
+import PIL
+
+from zenml.materializers.pillow_image_materializer import (
+    PillowImageMaterializer,
+)
+from zenml.pipelines import pipeline
+from zenml.steps import step
+
+JPEG_FILE_PATH = os.path.join(
```

Comment:
feels a bit weird to load an image from examples/ here, I would suggest to create a random image at the start of the test instead and try to load that. E.g. you could use something like this to create a random image: https://stackoverflow.com/questions/10901049/create-set-of-random-jpgs

Reply:
Thanks, yes I had the same idea overnight. No need to load an image at all, though this did reveal a potential bug that I've also now fixed. (TL;DR: if you create an image de novo in Pillow, it doesn't come with an inbuilt image format (vs if you load one from a file) so actually that didn't work with our materializer.. It now does).